neko/docker-compose.yml

```
@@ -0,0 +1,23 @@
+version: '3.7'
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: neko_web_1
+      APP_PORT: 8080
+      PROXY_AUTH_ADD: "false"
+
+  web:
+    image: ghcr.io/m1k1o/neko/firefox:3.0.3@sha256:9b5d3e04af1024a44c2aa8938c78f155d64b0ab9459feb424c34b461155a3f98
+    environment:
+      PUID: 1000
+      PGID: 1000
+      NEKO_DESKTOP_SCREEN: 1920x1080@30
+      NEKO_MEMBER_MULTIUSER_USER_PASSWORD: neko
+      NEKO_MEMBER_MULTIUSER_ADMIN_PASSWORD: umbrel

```

Maybe use the deterministic password here.
```suggestion
      NEKO_MEMBER_MULTIUSER_ADMIN_PASSWORD: $APP_PASSWORD

```